prime-router/metadata/schemas/NM/nm-covid-19-csv.schema

```diff
@@ -269,11 +269,11 @@ elements:
       - name: Pregnant
         format: $alt
     altValues:
-      - code: Y
+      - code: 77386006
         display: Y
-      - code: N
+      - code: 60001007
         display: N
-      - code: UNK
+      - code: 261665006
         display: U
```

Comment:
Bug fix:   The "code" values (Y, N, UNK) were not correct.  They need to match to the weird number values in the valueset.   I think we don't enforce that right now, so we only find out at runtime when unknown code values appear, eg, as generated by the fake data generator.

Reply:
These ALT values match the real LOINC values. Why do we need them? Suggest you change the format to be $code? Or is there something else that I'm missing?

Reply:
The valueset we have in covid-19.schema has Pregnant, Not Pregnant, and Unknown.
These appear to be the correct LOINC codes.  See https://loinc.org/82810-3/

So, New Mexico appears to want Y, N, UNK, which does not appear to be the LOINC enumeration.   Hence the need for an $alt valueset.

Reply:
ok